### PR TITLE
Pyic:4854 : Abandon journey

### PR DESCRIPTION
**Pull request by shivanshuit914**
## Proposed changes

- Create an abandon journey page
- Redirect user to appropriate page based on their selection

### What changed

- Created new page for the abandon journey
- Created routing for the abandon journey
- Welsh translation
- Browser tests

### Why did it change

- To allow user to select different CRI to prove their identity.

### Issue tracking

- [PYIC-4854](https://govukverify.atlassian.net/browse/PYIC-4854)

[PYIC-4854]: https://govukverify.atlassian.net/browse/PYIC-4854?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: tests/unit/src/app/kbv/controllers/prove-identity-another-way.test.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+      expect(req.axios.post).toHaveBeenCalledWith(ABANDON, undefined, {
+        headers: {
+          "session-id": req.session.tokenId,
+          session_id: "session-id",
+        },
+      });
+    });
+
+    it("should not call abandon endpoint", async () => {
```
Comment (blakeyp) [inline tests/unit/src/app/kbv/controllers/prove-identity-another-way.test.js:42]: We probably want `describe`s here? one "when radio value is stop" the other "when radio value is continue"

**Review by blakeyp — COMMENTED** [review]
Looks good to me ✅

File: src/app/kbv/steps.js [hunk 2]
```diff
@@ -34,6 +35,31 @@ module.exports = {
...
+        value: "stop",
+        next: "/oauth2/callback",
+      },
+      {
+        field: "abandonRadio",
+        value: "continue",
+        next: [
+          {
+            fn: proveIdentityAnotherWayController.prototype
+              .isQuestionJourneyStarted,
```
Comment (blakeyp) [inline src/app/kbv/steps.js:55]: Would it be simpler and less brittle if we just use the existing `loadQuestionController.prototype.hasQuestion` method here instead - if there's a question in the session then they've been to a question page, if there's no question they haven't got past the start page yet. Thoughts @shivanshuit914 @merlinc?

Comment (blakeyp) [reply src/app/kbv/steps.js:55]: The only edge case is if they've been through the start page to a question and then gone browser back to the start page (but we don't have back links so we're not really supporting 'back')

**Review by Caitlin-cooling — DISMISSED** [review]
Meets the ACs and test coverage is good! I like the use of the load-question controller for routing 👍

**Review by merlinc — APPROVED** [review]
I'm not thrilled about the block involving `const choice = req.form.values.abandonRadio` - however there isn't an immediately clear solution, so I'll approve this, and we can fix forward, protected by the tests that have been added here.

**Pull request merged**